messages/en.json

```
@@ -2,5 +2,7 @@
   "store/place-order-button": "Place Order",
   "store/checkout-product-count": "{quantity, plural, one {# product} other {# products}}",
   "store/edit-cart-label": "Edit Cart",
-  "store/view-cart-label": "View Cart"
+  "store/view-cart-label": "View Cart",
+  "store/place-order.close-error-alert-label": "Close",
+  "store/place-order.generic-error-message": "The transaction can't be completed. Please try again soon."
```

Shouldn't this be in past tense?

---

I "extracted" this message directly from figma 😅. you suggest to change it to something like the following?

```suggestion
  "store/place-order.generic-error-message": "The transaction couldn't be completed. Please try again soon."
```

---

Yes, I _think_ that sounds a bit more natural, but if that's how it is on Figma then I we can keep it like this I guess.

---

we could ask Vlad to review the english translations as well when the time comes for the translations reviews